lethal-squirrels/workforce-management
# Delete a Computer

## REFERENCE

Ticket: #8 

## DESCRIPTION
Changed the computer view so the fields appear in correct order
Updated the computer controller so that only unassigned computers can be deleted

## STEPS TO TEST
1. Clone aj to a local repo
2. Test functionality
	Click on Computers.
        You should see a list of computers
       Add a new computer
        Try and delete that computer
      You should be able to delete it
      Now try and delete a computer that is assigned to an employee
      You should not be able to delete it
